### ISSUE TITLE
How to enforce the mine and role permission

### ISSUE THREAD

**Issue by carolin-skiply** [directus/app#1066]
Hi,
I am trying to enforce the mine and role permission (see [here](https://docs.directus.io/app/admin/permissions.html#collection-level)) without success.

I created through the directus interface a field created_by which is a directus user.
![image](https://user-images.githubusercontent.com/34483024/48008487-14d98880-e11a-11e8-826d-a94a2547584b.png)

For one of my entry, I set the field created_by to an existing user.
So then I created a new role and tried to set the permission to role or mine but these options are not available.
Did I miss something ?

**Comment by benhaynes** [comment]
Hmm, @rijkvanzanten any thoughts on this? Is `created_on` also required to enable `mine` and `role`?

**Comment by benhaynes** [comment]
Hey @carolin-skiply – I am not able to reproduce this one. Here are the steps I took (7.0.5 Build):

1. Added a User interface ("User Created" Field Type) named `created_by` to Collection
2. Created a few items within Collection with the Admin User
3. Added a new role called "Member"
4. Went to Settings > Permissions
5. Clicked on the READ/UPDATE for the Collection
6. See options for Mine/Role – set to "Mine"
7. Created a new user with Member role
8. Logged in as that User (Member Role)
9. Did **not** see any of the Admin's items
10. Created a new item... and confirmed it set the Created By to User/Member

So everything seems to be working... can you send some specific steps you followed and/or a schema to reproduce?

**Comment by carolin-skiply** [comment]
Hi @benhaynes,
I tried following your steps :
1- So I add the field created_by in my collection (so_boitier) through your web interface
Here are the database screenshots :
![image](https://user-images.githubusercontent.com/34483024/48190043-e807ba00-e341-11e8-9669-6d704e855e0c.png)
![image](https://user-images.githubusercontent.com/34483024/48190100-138aa480-e342-11e8-85ba-dea8ff604964.png)
but nothing in directus_relations. (is that normal ?)

2- I created with my account some items
3- Added a new role but permission mine/role are still not available.

I stopped there.

**Comment by carolin-skiply** [comment]
Hi,
I've made progress :)

I started fresh on a new install and followed the steps you gave me.
So now, I can see in permission the 4 options : None / Mine / Role / Full
So I chose Role and login with my user with Member Role. However, the rabbit is running but nothing seems to appear (in my admin view I set at least one existing entry to my user).

Also I got this error in my console : 
![image](https://user-images.githubusercontent.com/34483024/49215087-474c6d80-f3c8-11e8-8cf7-6e63b37f64cd.png)
